tests/Security/Voter/OrganizationVoterTest.php

```
@@ -0,0 +1,208 @@
+        ]);
+
+        $this->assertSame(VoterInterface::ACCESS_GRANTED, $result);
+    }
+
+    public function testHandlesMultipleAttributesWithOneDeniedReturnsGranted(): void
+    {
+        // Symfony Voter grants if AT LEAST ONE attribute returns true
+        $token = $this->createMockToken($this->user);
+
+        $this->permissionService->method('hasOrganizationPermission')
+            ->willReturnCallback(function (User $user, Organization $org, string $permission): bool {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)